spec/scrub/auto_scrub_spec.rb

```diff
@@ -1,14 +1,8 @@
 # frozen_string_literal: true
 
-require "spec_helper"
 require "scrub/autoscrub"
 
 RSpec.describe Scrub::AutoScrub do
-  before(:each) do
```

Comment:
It was unclear to me whether the specs will hit `OCLC_URL` now or not, but I'm thinking they still would when running in Github Actions (i.e. we aren't committing the cached max_ocn, right?) If so, it might be worth continuing to stub this request in the specs.

Reply:
I don't understand what your concern is here.
The stub is moved to `spec_helper.rb:85`, and it seems to be working just fine?
And no, we are not committing the max ocn.

Reply:
I missed that the stub was moved to `spec_helper`, so I think it's fine as-is. Without the stub, I was concerned that 1) we introduce an external dependency into the specs and 2) every time we run the specs in github it would hit `OCLC_URL` for real. But we still have the stub, so it doesn't matter.

Reply:
Actually, I'm not seeing it in `spec_helper` - missed committing it?